runtime/westend/constants/src/lib.rs

```diff
@@ -96,6 +96,27 @@ pub mod fee {
 	}
 }
 
+/// System Parachains.
+pub mod system_parachain {
+	use xcm::latest::prelude::*;
+
+	/// Westmint parachain ID.
+	pub const WESTMINT_ID: u32 = 1000;
+	/// Collectives parachain ID.
+	pub const COLLECTIVES_ID: u32 = 1001;
+	/// BridgeHub parachain ID.
+	pub const BRIDGE_HUB_ID: u32 = 1002;
+
+	frame_support::match_types! {
+		pub type SystemParachain: impl Contains<MultiLocation> = {
```

Comment:
Type should be plural to fit in with the rest:
```suggestion
		pub type SystemParachains: impl Contains<MultiLocation> = {
```

Reply:
(I've already changed the companion to assume that, so hopefully it will build clean)